### PR TITLE
Added cucumberOptions

### PR DESCRIPTION
**Pull request by Shobika**
CucumberOptions - dryRun, monochrome and strict properties were added

### CODE REVIEW

File: content/cucumber/api.md [hunk 1]
```diff
@@ -772,6 +772,53 @@ import org.junit.runner.RunWith;
...
+
+```java
+package mypackage;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(dryRun=true)
```
Comment (mlvandijk) [inline content/cucumber/api.md:785]: Please use a consistent formatting for the specified option for each of the three examples, either with or without spaces around the `=` sign.

File: content/cucumber/api.md [hunk 2]
```diff
@@ -772,6 +772,53 @@ import org.junit.runner.RunWith;
...
+public class RunCukesTest {
+}
+```
+dryRun default value is false.
```
Comment (mlvandijk) [inline content/cucumber/api.md]: Please use a whole sentence starting with a capital letter, optionally marking the option and it's value, for instance: "The default option for `dryRun` is `false`. (Note: please do this consistently for all three options)

File: content/guides/browser-automation.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ public class ExampleSteps {
         element.submit();
    }
 
-   @Then("^ the page title should start with \"(.*)\"$")
+   @Then("^the page title should start with \"(.*)\"$")
```
Comment (mlvandijk) [inline content/guides/browser-automation.md:61]: Nice catch, thanks!

**Review by mlvandijk — CHANGES_REQUESTED** [review]
Thank you for this contribution! 
Just a few very minor edit requests for consistency only. Please let me know if you are ok to do these yourself (otherwise I'm happy to do them myself).

**Comment by Shobika** [comment]
Thank you for your feedback..I will do those changes myself and send a request again.

**Comment by Shobika** [comment]
@mlvandijk finished updating.

**Review by mlvandijk — APPROVED** [review]

**Comment by aslakhellesoy** [comment]
Hi @Shobika,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by mlvandijk** [comment]
Thanks for your contribution @Shobika !

**Comment by olleolleolle** [comment]
Thanks, Shobika for this! You're improving everyone's documentation experience!

**Pull request merged**